lib/attachments/gridstore-storage.js

```diff
@@ -33,6 +33,23 @@ class GridstoreStorage {
         });
     }
 
+    updateFileWithContentHashMetadata(args, callback, hash, calculatedFileContentHash) {
```

Comment:
The convention for callbacks is to always use `callback` as the last function argument.

```
updateFileWithContentHashMetadata(args, hash, calculatedFileContentHash, callback)
```